crypto/rand/rand_local.h

```diff
@@ -308,8 +308,10 @@ struct rand_drbg_st {
     size_t seedlen;
     DRBG_STATUS state;
 
+#ifndef FIPS_MODE
     /* Application data, mainly used in the KATs. */
```

Comment:
Doesn't this comment cause alarm?

Reply:
I was wondering about that, but having looked around, I only saw that used in test programs (i.e applications), so. 

They can always be re-enabled selectively later on, if actually needed.